src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/controller/PersonControllerIntTest.kt

```diff
@@ -40,9 +38,6 @@ class PersonControllerIntTest() : IntegrationTestBase() {
   private var minimumPerson: Person? = null
   private var maximumPerson: Person? = null
 
-  @Autowired
-  lateinit var personRepository: PersonRepository
-
```

Comment:
starting to move towards clearing this out after each test and using JPA to populate rather than the SQL files - one step at a time!